spec/mail/securityconsiderations.mdown

```diff
@@ -10,7 +10,32 @@ Automatic conversion of charsets that allow hidden channels for ASCII text, such
 
 To allow the client to restrict the volume of data it can receive in response to a request, a maximum length may be requested for the data returned for a textual body part. However, truncating the data may change the semantic meaning, for example truncating a URL changes its location. Servers that scan for links to malicious sites should take care to either ensure truncation is not at a semantically significant point, or to rescan the truncated value for malicious content before returning it.
 
-## Email Submission
+## HTML email display
+
+HTML message bodies provide richer formatting for emails but present a number of security challenges especially when embedded in a webmail context in combination with interface HTML. Clients that render HTML email should make careful consideration of the potential risks, including:
+
+* If JavaScript is allowed to execute it can rewrite email to change its
+  content on subsequent opening, allowing users to be mislead. In webmail systems, if run in the same origin as the interface it can access and exfiltrate all user content accessible via the browser, including all other emails and potentially contacts, calendar events, settings, and credentials. It can also rewrite the interface to undetectably phish passwords. A compromise is likely to be persistent, not just for the duration of page load, due to exfiltration of session credentials or installation of a service worker that can intercept all subsequent network requests (this however would only be possible if blob downloads are also available on the same origin, and the service worker script is attached to the message).
+* HTML documents may load content directly from the internet, rather than just
+  referencing attached resources. For example you may have an `<img>` tag with an external `src` attribute. This may leak to the sender when a message is opened, as well as the IP address of the recipient. Cookies may also be sent and set by the server, allowing tracking between different emails and even website visits and advertising profiles.
```

Comment:
I'd recommend adding a [CORS settings attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes) to those tags: `crossorigin="anonymous"` will tell the browser not to use cookies.

Referrer information can be stripped with a [referer policy](https://www.w3.org/TR/referrer-policy/) (as you suggest below)